dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/apps/jaxrspropagation/JaxRsEndpoints.java

```diff
@@ -210,6 +215,28 @@ public Response readSpansMpTel11(@Context UriInfo uriInfo, @PathParam("traceId")
         return Response.ok(TEST_PASSED).build();
     }
 
+    //Gets a list of spans created by open telemetry when a test was running and confirms the spans are what we expected and IDs are propagated correctly
+    //spanExporter.reset() should be called at the start of each new test.
+    @GET
+    @Path("/readspanswithspan/{traceId}")
+    public Response readSpansWithSpan(@Context UriInfo uriInfo, @PathParam("traceId") String traceId) {
+        List<SpanData> spanData = spanExporter.getFinishedSpanItems(2, traceId);
```

Comment:
Note that this class is not a test servlet so it doesn't nicely report errors and you have to go fishing in the log to find out what broke.

In future, don't add more tests to this class.